modules/log6x-logging-loki-cli-install.adoc

```diff
@@ -0,0 +1,177 @@
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `Subscription` object for {loki-op}:
++
+.Example `Subscription` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: loki-operator
+  namespace: openshift-operators-redhat # <1>
+spec:
+  channel: stable # <2>
+  name: loki-operator
+  source: redhat-operators # <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+<2> Specify `stable`, or `stable-6.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object you created when you configured the Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `namespace` object for the {clo}:
++
+.Example `namespace` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging # <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true" # <2>
+----
+<1> The Red{nbsp}Hat OpenShift Logging Operator is only deployable to the `openshift-logging` namespace.
+<2> A string value that specifies the label as shown to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+. Apply the `namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create an `OperatorGroup` object
++
+.Example `OperatorGroup` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  targetNamespaces:
+  - openshift-logging
+----
+<1> You must specify the `openshift-logging` namespace.
+
+. Apply the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `Subscription` object:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  channel: stable # <2>
+  name: cluster-logging
+  source: redhat-operators # <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify `stable`, or `stable-6.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+
```

Comment:
We are missing the part about creating a LokiStack.

Reply:
added